### PR TITLE
Replace support for SemanticForms with PageForms

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

**Comment by mwjames** [comment]
Returns with 
```
PHP Fatal error:  Uncaught exception 'Exception' with message '/home/travis/build/SemanticMediaWiki/mw/extensions/PageForms/extension.json does not exist!' in /home/travis/build/SemanticMediaWiki/mw/includes/registration/ExtensionRegistry.php:106
Stack trace:
#0 /home/travis/build/SemanticMediaWiki/mw/includes/GlobalFunctions.php(181): ExtensionRegistry->queue('/home/travis/bu...')
#1 /home/travis/build/SemanticMediaWiki/mw/LocalSettings.php(122): wfLoadExtension('PageForms')
#2 /home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php(83): require('/home/travis/bu...')
#3 /home/travis/build/SemanticMediaWiki/mw/maintenance/update.php(214): require_once('/home/travis/bu...')
#4 {main}
  thrown in /home/travis/build/SemanticMediaWiki/mw/includes/registration/ExtensionRegistry.php on line 106
Fatal error: Uncaught exception 'Exception' with message '/home/travis/build/SemanticMediaWiki/mw/extensions/PageForms/extension.json does not exist!' in /home/travis/build/SemanticMediaWiki/mw/includes/registration/ExtensionRegistry.php on line 106
Exception: /home/travis/build/SemanticMediaWiki/mw/extensions/PageForms/extension.json does not exist! in /home/travis/build/SemanticMediaWiki/mw/includes/registration/ExtensionRegistry.php on line 106
```

**Comment by paladox** [comment]
Hi yes, because I have updated it three times so that test is the old one, still has to run 2 more times.

**Comment by mwjames** [comment]
Okay, this were the dog bites tail, packagist isn't updated yet (because this is the PR that changes it) therefore when trying to fetch it via Composer it will fail with above message.

**Comment by paladox** [comment]
@mwjames oh yep, I wonder how we can get composer to update without doing init?

**Comment by mwjames** [comment]
I'm being adventurous and merge this without testing it because I can't.

**Comment by paladox** [comment]
I guess I could create a separate branch on my fork and just copy over the branch patch-20 and update the script to check if it will work

**Comment by mwjames** [comment]
Just merged this, would be nice if someone can confirm this still works @Seb35!

**Comment by paladox** [comment]
Thanks, @mwjames do you have access to packagist?

**Comment by paladox** [comment]
@mwjames seems to pace though I did hack around it so I am not 100% sure it will pass when packagist is updated, see https://travis-ci.org/paladox/SemanticFormsSelect/jobs/177265828

**Comment by paladox** [comment]
@mwjames see https://travis-ci.org/paladox/SemanticFormsSelect/jobs/177272059 :)

**Comment by paladox** [comment]
Just needs someone to update packagist now.

File: SemanticFormsSelect.php [hunk 1]
```diff
@@ -68,11 +68,11 @@ public static function initExtension() {
 	public static function onExtensionFunction() {
 
 		if ( !defined( 'SF_VERSION' ) ) {
-			die( '<b>Error:</b><a href="https://github.com/SemanticMediaWiki/SemanticFormsSelect/">Semantic Forms Select</a> requires the <a href="https://www.mediawiki.org/wiki/Extension:SemanticForms">Semantic Forms</a> extension. Please install and activate this extension first.' );
+			die( '<b>Error:</b><a href="https://github.com/SemanticMediaWiki/SemanticFormsSelect/">Semantic Forms Select</a> requires the <a href="https://www.mediawiki.org/wiki/Extension:PageForms">Semantic Forms</a> extension. Please install and activate this extension first.' );
```
Comment (JeroenDeDauw) [inline SemanticFormsSelect.php]: Still says Semantic Forms

Comment (paladox) [inline SemanticFormsSelect.php]: Hi yep missed that one, that was fixed by @mwjames in https://github.com/SemanticMediaWiki/SemanticFormsSelect/commit/483f3702a7daa9f3b98a6f9b7daa2288b6c8978e

**Pull request merged**